src/shared/components/common/data-type-select.tsx

```diff
@@ -31,33 +35,41 @@ export class DataTypeSelect extends Component<
 
   render() {
     return (
-      <div className="data-type-select btn-group btn-group-toggle flex-wrap">
+      <div
+        className="data-type-select btn-group btn-group-toggle flex-wrap"
+        role="group"
+      >
+        <input
+          id={`${this.id}-posts`}
+          type="radio"
+          className="btn-check"
+          value={DataType.Post}
+          checked={this.state.type_ == DataType.Post}
```

Comment:
Use triple equals.

Reply:
Yeah there are a loooooot of these everywhere, I was tempted to replace them everywhere in all the files I touched here.